packages/renderer/src/AppNavigation.svelte

```diff
@@ -44,7 +48,7 @@ function clickSettings(b: boolean) {
 <nav
   class="group w-leftnavbar min-w-leftnavbar flex flex-col hover:overflow-y-none bg-[var(--pd-global-nav-bg)] border-[var(--pd-global-nav-bg-border)] border-r-[1px]"
   aria-label="AppNavigation">
-  <NavItem href="/" tooltip="Dashboard" bind:meta={meta}>
+  <NavItem href="/" tooltip="Dashboard" bind:meta={meta} bind:tooltipHidden={tooltipHidden}>
```

Comment:
hello, why are we using bind: prefix for tooltipHidden ?

Reply:
this file has the event listener for when a context menu is closed, so binding the value of `tooltipHidden` allows each `NavItem` keep track of their tooltip (show or hide) without having multiple event listeners. I can remove this binding, but it will require adding lines 33-35 to `NavItem` instead of here